packages/react/src/components/ContextualMenu/ContextualMenu.test.tsx

```diff
@@ -1003,9 +1013,10 @@ describe('ContextualMenu', () => {
       buttonRef.current!.openMenu();
       buttonRef.current!.dismissMenu();
       const tree = component.toJSON();
+
       expect(tree).toMatchSnapshot();
 
-      ReactDOM.createPortal = createPortal;
+      jest.resetAllMocks();
```

Comment:
This is also not a reliable cleanup method if the test fails and should be moved to afterEach